frontend/src/components/table/TableBase.tsx

```diff
@@ -222,7 +224,11 @@ const TableBase = <T,>({
   return (
     <>
       {(toolbarContent || showPagination) && (
-        <Toolbar inset={{ default: 'insetNone' }} customChipGroupContent={<></>}>
+        <Toolbar
+          inset={{ default: 'insetNone' }}
+          className="pf-v5-u-w-100"
```

Comment:
why was this needed? `className="pf-v5-u-w-100"`

Reply:
Without the width of 100% added here, when there are no actions next to the filter input, the pagination for the table gets pushed to the immediate left of the filter input instead of floating to the far right as intended.